lua/package-info/helpers/get_dependency_name_from_line.lua

```diff
@@ -6,6 +6,10 @@ local clean_version = require("package-info.helpers.clean_version")
 -- @param value: string - value to check if conforms
 -- @return boolean
 local is_valid_dependency_version = function(value)
+    if string.find(value, "catalog:") ~= nil then
+        return true
+    end
```

Comment:
Move this to a fn above called is_pnpm_version or something similar since string find for catalog is a bit ambiguous